### ISSUE TITLE
Unable to Work Preprocessing Function Parameter for model.train()

### ISSUE THREAD

**Issue by jjohn485** [divamgupta/image-segmentation-keras#386]
Hi, I've been working with your segmentation classifier and it has been working great so far! 
My only issue so far is that I am trying to normalize my image training set and I haven't been able to find a way to do it successfully yet. 
I have tried both an ImagerGenerator and feeding it a directory full of numpy files and it doesn't want to take them. 

I've been wanting to use the preprocessing parameter but I am unable to discern how to use it properly. I didn't see any resources talking about it either. 

Any help would be greatly appreciated!

**Comment by jjohn485** [comment]
Figured it out!
